### ISSUE TITLE
Question of the performance on SIDD validation dataset

### ISSUE THREAD

**Issue by alwaysuu** [Reyhanehne/CVF-SID_PyTorch#8]
Thanks for sharing the source code and the pre-trained model on SIDD validation dataset. I tested the provided pre-trained model on SIDD validation dataset and only got 34.14/0.855 (SPNR/SSIM).
![image](https://user-images.githubusercontent.com/48625631/219932655-ac50bef5-8f90-42e6-b253-ab435212abab.png)

Moreover, I tried to train the denoiser ( CVF-SID (S) ) on SIDD validation dataset directly, and I can only get 34.05/0.847.

I am using the same versions of python and pytorch as yours. I would like to know if I missed something important.

**Comment by Reyhanehne** [comment]
Please make sure that you trained the pre-trained model on the SIDD-Medium dataset not the SIDD-Small dataset.

**Comment by alwaysuu** [comment]
Thanks for the reply. 
First, I would like to know why I cannot get the performance as yours by using the provided pre-trained model on SIDD validation dataset. Do I need to continue training the pre-trained model to get better performance?
Second, when I follow the procedure of CVF-SID (S) and directly train the model on the testset (SIDD validation), I also cannot get the performance as yours. I think the way you mentioned "make sure that you trained the pre-trained model on the SIDD-Medium dataset" should be CVF-SID (T)?
![image](https://user-images.githubusercontent.com/48625631/220612574-c6373033-2a61-4591-8237-f8ce8f203c82.png)

**Comment by lulu-165** [comment]
Hello, may I ask what the first and second dimensions in the data set respectively mean?
![image](https://user-images.githubusercontent.com/38996467/235044951-0a8e463f-aa94-40a6-9ce5-149e8c569ca4.png)

**Comment by alwaysuu** [comment]
Hello, it means that the validation set has 40 images with 32 blocks, each of which has a size (256,256,3)

**Comment by Reyhanehne** [comment]
Hello, 
Please refer to Section 4.1 of the main paper.  
"Validation and benchmark splits are used, each containing 32 patches of size 256 × 256 from 40 images".

**Comment by luckly41** [comment]
> 你好，请问数据集中的第一维度和第二维度分别是什么意思？ ![image](https://user-images.githubusercontent.com/38996467/235044951-0a8e463f-aa94-40a6-9ce5-149e8c569ca4.png)

Can I have your WeChat account? I am also working on this project

**Comment by lulu-165** [comment]
ok,please&nbsp; give me your number.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Reyhanehne/CVF-SID_PyTorch"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年5月1日(星期一) 中午12:56
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Reyhanehne/CVF-SID_PyTorch] Question of the performance on SIDD validation dataset (Issue #8)





  
你好，请问数据集中的第一维度和第二维度分别是什么意思？ 
  
Can I have your WeChat account? I am also working on this project

**Comment by luckly41** [comment]
15524828279，thank you




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Reyhanehne/CVF-SID_PyTorch"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年5月1日(星期一) 中午12:59
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Reyhanehne/CVF-SID_PyTorch] Question of the performance on SIDD validation dataset (Issue #8)





 ok,please&amp;nbsp; give me your number. 
 
 
 
 
 ------------------&amp;nbsp;原始邮件&amp;nbsp;------------------ 
 发件人:                                                                                                                        "Reyhanehne/CVF-SID_PyTorch"                                                                                    ***@***.***&amp;gt;; 
 发送时间:&amp;nbsp;2023年5月1日(星期一) 中午12:56 
 ***@***.***&amp;gt;; 
 ***@***.******@***.***&amp;gt;; 
 主题:&amp;nbsp;Re: [Reyhanehne/CVF-SID_PyTorch] Question of the performance on SIDD validation dataset (Issue #8) 
 
 
 
 
 
    
 你好，请问数据集中的第一维度和第二维度分别是什么意思？  
    
 Can I have your WeChat account? I am also working on this project

**Comment by xuezhilin** [comment]
> Thanks for sharing the source code and the pre-trained model on SIDD validation dataset. I tested the provided pre-trained model on SIDD validation dataset and only got 34.14/0.855 (SPNR/SSIM). ![image](https://user-images.githubusercontent.com/48625631/219932655-ac50bef5-8f90-42e6-b253-ab435212abab.png)
> 
> Moreover, I tried to train the denoiser ( CVF-SID (S) ) on SIDD validation dataset directly, and I can only get 34.05/0.847.
> 
> I am using the same versions of python and pytorch as yours. I would like to know if I missed something important.
Can I ask if your issue has been resolved? Have you been able to replicate the results from the paper?

**Comment by luckly41** [comment]
可以加微信聊，我也只是复制出结果，还没换数据集



---Original---
From: ***@***.***&gt;
Date: Sat, May 20, 2023 16:32 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [Reyhanehne/CVF-SID_PyTorch] Question of the performance on SIDDvalidation dataset (Issue #8)




  
Thanks for sharing the source code and the pre-trained model on SIDD validation dataset. I tested the provided pre-trained model on SIDD validation dataset and only got 34.14/0.855 (SPNR/SSIM). 
 
Moreover, I tried to train the denoiser ( CVF-SID (S) ) on SIDD validation dataset directly, and I can only get 34.05/0.847.
 
I am using the same versions of python and pytorch as yours. I would like to know if I missed something important.
 Can I ask if your issue has been resolved? Have you been able to replicate the results from the paper?

**Comment by xuezhilin** [comment]
好啊，我的微信二维码在附件中。




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Reyhanehne/CVF-SID_PyTorch"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年5月20日(星期六) 下午4:36
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Reyhanehne/CVF-SID_PyTorch] Question of the performance on SIDD validation dataset (Issue #8)





 可以加微信聊，我也只是复制出结果，还没换数据集 
 
 
 
 ---Original--- 
 From: ***@***.***&amp;gt; 
 Date: Sat, May 20, 2023 16:32 PM 
 To: ***@***.***&amp;gt;; 
 Cc: ***@***.******@***.***&amp;gt;; 
 Subject: Re: [Reyhanehne/CVF-SID_PyTorch] Question of the performance on SIDDvalidation dataset (Issue #8) 
 
 
 
 
    
 Thanks for sharing the source code and the pre-trained model on SIDD validation dataset. I tested the provided pre-trained model on SIDD validation dataset and only got 34.14/0.855 (SPNR/SSIM).  
   
 Moreover, I tried to train the denoiser ( CVF-SID (S) ) on SIDD validation dataset directly, and I can only get 34.05/0.847. 
   
 I am using the same versions of python and pytorch as yours. I would like to know if I missed something important. 
  Can I ask if your issue has been resolved? Have you been able to replicate the results from the paper?

**Comment by luckly41** [comment]
好像看不到附件，wild200041你加我吧



---Original---
From: ***@***.***&gt;
Date: Sat, May 20, 2023 17:23 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [Reyhanehne/CVF-SID_PyTorch] Question of the performance on SIDDvalidation dataset (Issue #8)




 好啊，我的微信二维码在附件中。 
 
 
 
 
 ------------------&amp;nbsp;原始邮件&amp;nbsp;------------------ 
 发件人:                                                                                                                        "Reyhanehne/CVF-SID_PyTorch"                                                                                    ***@***.***&amp;gt;; 
 发送时间:&amp;nbsp;2023年5月20日(星期六) 下午4:36 
 ***@***.***&amp;gt;; 
 ***@***.******@***.***&amp;gt;; 
 主题:&amp;nbsp;Re: [Reyhanehne/CVF-SID_PyTorch] Question of the performance on SIDD validation dataset (Issue #8) 
 
 
 
 
 
  可以加微信聊，我也只是复制出结果，还没换数据集  
   
   
   
  ---Original---  
  From: ***@***.***&amp;amp;gt;  
  Date: Sat, May 20, 2023 16:32 PM  
  To: ***@***.***&amp;amp;gt;;  
  Cc: ***@***.******@***.***&amp;amp;gt;;  
  Subject: Re: [Reyhanehne/CVF-SID_PyTorch] Question of the performance on SIDDvalidation dataset (Issue #8)  
   
   
   
   
      
  Thanks for sharing the source code and the pre-trained model on SIDD validation dataset. I tested the provided pre-trained model on SIDD validation dataset and only got 34.14/0.855 (SPNR/SSIM).   
     
  Moreover, I tried to train the denoiser ( CVF-SID (S) ) on SIDD validation dataset directly, and I can only get 34.05/0.847.  
     
  I am using the same versions of python and pytorch as yours. I would like to know if I missed something important.  
   Can I ask if your issue has been resolved? Have you been able to replicate the results from the paper?

**Comment by sxdyyds** [comment]
> 感谢您分享 SIDD 验证数据集上的源代码和预训练模型。我在 SIDD 验证数据集上测试了提供的预训练模型，只得到了 34.14/0.855 （SPNR/SSIM）。 ![image](https://user-images.githubusercontent.com/48625631/219932655-ac50bef5-8f90-42e6-b253-ab435212abab.png)
> 
> 此外，我尝试直接在 SIDD 验证数据集上训练降噪器 （ CVF-SID （S） ），我只能得到 34.05/0.847。
> 
> 我正在使用与您相同版本的 python 和 pytorch。我想知道我是否错过了什么重要的东西。

You should test it on SIDD benchmark, not SIDD validation dataset. Please see table 1 on block bold words.
